### DISCUSSION TITLE
sftp connection through proxy

### DISCUSSION THREAD

**Discussion by tranle** [iterate-ch/cyberduck#14453] [Q&A]
Is there a way to setup cyberduck for sftp through a proxy ?
Thanks.

**Comment by AliveDevil** [comment]
Depending on what you want to achieve, you can configure a SSH jump host/proxyjump/bastion server ([SFTP-Documentation](https://docs.cyberduck.io/protocols/sftp/#connect-via-ssh-tunnel-through-bastion-server)), or if you want to proxy the connection, you can configure Cyberduck to use the [system proxy](https://docs.cyberduck.io/cyberduck/connection/#proxy-configuration).

The configured FTP proxy is used for SFTP connections as well (in Cyberduck).
